src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/IncidentQuestion.kt

```diff
@@ -31,11 +32,12 @@ data class IncidentQuestion(
   @EmbeddedId
   val id: IncidentQuestionId,
 
-  @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Removing lazy loading for the ManyToOne (and others in this PR) reduced the number of selects (for one of the biggest queries) by over 250